### PR TITLE
Update Bazaar to version 2.7

### PR DESCRIPTION
**Pull request by fmccann**
Updated bzr to latest version.

### CODE REVIEW

**Comment by DomT4** [comment]
Please see the discussion in https://github.com/Homebrew/homebrew/pull/48849. Is this stable?

File: Library/Formula/bazaar.rb [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
+  url "https://launchpad.net/bzr/2.7/2.7.0/+download/bzr-2.7.0.tar.gz"
+  sha256 "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
 
   bottle do
     cellar :any_skip_relocation
-    sha256 "ad25ac7aae3262bb674c9e2261415daecfaaf9b5d5f57ff14428be9409b895b8" => :el_capitan
-    sha256 "29ff8d9c8fe3eee3a2a2172d5247970a68fa8d24c7516ae06bed9b3849259f2c" => :yosemite
-    sha256 "cfda2b2e1b18687428c407155cb2e497563279d861120d0aa3f5a9e886cabf76" => :mavericks
-    sha256 "7aa76616196f64b7a979708bb67703d9e70d4fe74bae55f204c9c844dfe71611" => :mountain_lion
+    sha256 "5a4a0979f70e18b606ef04b1cecbd32cbb4633d75c07b6980b2543cbcad5e7bf" => :el_capitan
```
Comment (DomT4) [inline Library/Formula/bazaar.rb]: Please don't touch the bottle block on updates. We handle that, Thanks :smiley:.

**Comment by fmccann** [comment]
Should I remove the block or leave it as is?

**Comment by DomT4** [comment]
Leave it as-is. CI knows to ignore it.

**Comment by fmccann** [comment]
With regard to #48849, yes this is the new stable version (https://launchpad.net/bzr/+milestone/2.7.0)

**Comment by DomT4** [comment]
> yes this is the new stable version 

Great! It was still trunk the other day, at least according to the release chain on the website.

CI disagrees with your checksum:
```
==> Verifying bazaar-2.7.0.tar.gz checksum
Error: SHA256 mismatch
Expected: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
Actual: 0d451227b705a0dd21d8408353fe7e44d3a5069e6c4c26e5f146f1314b8fdab3
```

**Comment by fmccann** [comment]
Am I calculating the sha wrong? This is what I'm seeing on my box whether I calculate manually or via brew create:

$> shasum -a 256 bazaar-2.7.0.tar.gz 
e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855  bazaar-2.7.0.tar.gz

**Comment by DomT4** [comment]
You're not calculating it wrong but I'm getting a checksum that agrees with CI locally:
```
~> shasum -a 256 bzr-2.7.0.tar.gz
0d451227b705a0dd21d8408353fe7e44d3a5069e6c4c26e5f146f1314b8fdab3  bzr-2.7.0.tar.gz
```

**Comment by fmccann** [comment]
This is gonna sound dumb, but I think I curl'd the file without redirects. I'm gonna update the sha.

$> curl -L -O "https://launchpad.net/bzr/2.7/2.7.0/+download/bzr-2.7.0.tar.gz"
$> shasum -a 256 bzr-2.7.0.tar.gz 
0d451227b705a0dd21d8408353fe7e44d3a5069e6c4c26e5f146f1314b8fdab3  bzr-2.7.0.tar.gz

**Comment by DomT4** [comment]
Ah, we've all done that from time to time!

You've accidentally dragged in a bunch of commits made to `master` today. Could you possibly rebase there to fix that? Makes examining the diff a lot easier without the noise.

**Comment by fmccann** [comment]
OK, should be rebased.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/c8fcc5cf36c63a4e98d82ef6c6075b8a71715de6. Thank you for your contribution to Homebrew @fmccann; we appreciate it! :smiley_cat:

**Pull request closed without merging**